OpenRA.Mods.Common/Traits/World/MapOptions.cs

```diff
@@ -24,6 +24,12 @@ public class MapOptionsInfo : ITraitInfo, ILobbyOptions, IRulesetLoaded
 		[Desc("Prevent the short game enabled state from being changed in the lobby.")]
 		public readonly bool ShortGameLocked = false;
 
+		[Desc("Default value of the bounty checkbox in the lobby.")]
+		public readonly bool BountyEnabled = true;
```

Comment:
Please create a new trait for these.  Bounties are very much a mod-specific thing, and shouldn't be forced on everyone else!

Reply:
yes, will do